sysid-application/src/integrationtest-generation/native/cpp/GenerationTest.cpp

```diff
@@ -163,6 +150,6 @@ TEST_F(GenerationTest, Drivetrain) {
 
   m_settings = sysid::kRomiConfig;
   m_manager.SaveJSON(m_jsonPath, 1, true);
-  wpi::outs() << "Testing: Romi Config\n";
+  fmt::print(stderr, "Testing: Romi Config\n");
   Run();
```

Comment:
Commenting this out would stop the Romi from being tested, then we can add it back once #114 is done